src/register/RegistrationPage.jsx

```diff
@@ -403,7 +403,7 @@ class RegistrationPage extends React.Component {
                 <ValidationFormGroup
                   for={field.name}
                   key={field.name}
-                  className="pt-10 mb-20 small"
+                  className="pt-10 mb-0 small"
```

Comment:
No need to add `mb-0` class, we need to keep `1rem` default `margin-bottom` set on `.form-group`.

Reply:
done